transaction.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3"
```

Comment:
How is this related to the blind bid?

Reply:
It's not. But I needed the types in order to be able to write all of the service messages.